demo/README.md

```diff
@@ -25,7 +25,29 @@ If you are working from the Release Package, these files have already been copie
 If you are working with source code, build the project and it will copy the required dependencies to the demo folder.
 
 ## Create Java keystore files
-See instructions in the [Secure Device Onboard Rendezvous Service README](https://github.com/secure-device-onboard/rendezvous-service/#generate-keystores). Once the keystore and truststore files are created, update docker-compose.yml to reflect the file name, path and password.  The default configured keystore is /certs/rendezvous-keystore.jks and the default configured truststore is /certs/rendezvous-truststore.jks. Default passwords for both: 123456
+The example IOT Platform SDK, Supply Chain Toolkit - Manufacturer and Supply Chain Toolkit - Reseller
+public key hash are whitelisted in the redis docker deployment. Please find the list of whitelisted hash and
+its corresponding SDO component.
```

Comment:
You may consider to rephrase this as below.
"The hashes for the default public keys present in iot-platform-sdk and supply-chain-tools are listed in following table."

Reply:
Done